releng/features/org.eclipse.ecf.filetransfer.httpclient5.feature/feature.xml

```diff
@@ -39,14 +39,7 @@
          unpack="false"/>
 
    <plugin
-         id="org.apache.commons.codec"
-         download-size="0"
-         install-size="0"
-         version="0.0.0"
-         unpack="false"/>
-
-   <plugin
-         id="org.apache.commons.logging"
+         id="org.apache.commons.commons-codec"
```

Comment:
Did `org.apache.commons.logging` get dropped on purpose here?

Reply:
Yes, because I checked and there are no requirements on this by any of the http5 stuff.  I suspect it was just copied from the 4.x stuff.   Here is SimRel after you contributed httpclient5:

![image](https://github.com/eclipse/ecf/assets/208716/714744b9-49ac-4a24-9095-395412c7ef99)

Note the image from the previous comment resolves using planner mode, so if something actually required it, even only transitively, it would still be present in the target for that location...